build_tools/kokoro/gcp_ubuntu/cmake-bazel/linux/x86-swiftshader/build.sh

```diff
@@ -36,37 +36,35 @@ python3 -c 'import jax; print(jax.__version__)'
 # Print SwiftShader git commit
 cat /swiftshader/git-commit
 
-# TODO(4195): TensorFlow integrations can only be built out-of-tree (and kokoro
-# does not support that).
-# Temporarily get around these conflicting restrictions by nesting the git
-# repo under `/src/github/iree/iree`
-echo "Nesting iree.git into /src/github/iree/iree"
-mv iree iree-core  # Temporarily move iree/ so that we move the repo under it.
-shopt -s dotglob nullglob  # Make globs include dotfiles/dirs like `.git`.
-mkdir iree
-mv * ./iree | true  # | true to catch "can't mv ./iree into itself"
-cd iree
-mv iree-core iree # Move iree-core/ back to iree/
-ls -1 -a
-
 echo "Initializing submodules"
 ./scripts/git/submodule_versions.py init
 
+# BUILD the integrations binaries with Bazel
+pushd integrations/tensorflow
+BAZEL_CMD=(bazel --noworkspace_rc --bazelrc=build_tools/bazel/iree-tf.bazelrc)
+BAZEL_BINDIR="$(${BAZEL_CMD[@]?} info bazel-bin)"
+"${BAZEL_CMD[@]?}" build --config=generic_clang //iree_tf_compiler:all
+popd
+
+CMAKE_BUILD_DIR="$HOME/iree/build/tf"
```

Comment:
Any particular reason to use this path instead of `../iree-build` or `../iree-build-tf` (just thinking about being consistent with some of our docs and other scripts)

Reply:
Yes. This is running in a docker container, so only mounted paths are writeable. As set up, we mount the directory in which IREE is checked out and the home directory, but not the parent of the directory in which IREE is checked out. See #4195

(as a side note, I hate sticking build directories as siblings to source directories)

Reply:
(technically we don't mount the real home directory, we create a fake one so it's also mounted on the local_ssd for fast IO, see https://github.com/google/iree/blob/main/build_tools/kokoro/gcp_ubuntu/docker_common.sh)